Tests/Functional/app/AppKernel.php

```diff
@@ -41,12 +41,17 @@ public function __construct($environment, $debug, $testCase = null)
      */
     public function registerBundles(): array
     {
-        return [
+        $bundles = [
             new FrameworkBundle(),
             new SecurityBundle(),
             new LexikJWTAuthenticationBundle(),
             new Bundle(),
         ];
+        if (class_exists(JoseFrameworkBundle::class)) {
+            $bundles[] = new JoseFrameworkBundle();
```

Comment:
If it's always installed, we could remove the class existence check

Reply:
I did that because the current CI includes PHP 7.1, which is not supported by Web-Token (unless out-of-life versions are used). 
This also explains why I excluded the tests in the composer scripts.